src/core/prometheus.cc

```diff
@@ -550,9 +553,27 @@ class metric_aggregate_by_labels {
         }
         std::unordered_map<std::map<sstring, sstring>, seastar::metrics::impl::metric_value>::iterator i = _values.find(labels);
         if ( i == _values.end()) {
+            _count.emplace(labels, 1);
             _values.emplace(std::move(labels), m);
         } else {
-            i->second += m;
+            switch (_aggregate_function) {
```

Comment:
it might be worthy to add an alias of `i->second` ? like
```c++
auto value& = i->second;
```
so we can just reference it here. as before this change, the function is a tiny one, where `i` would suffice without hurting the readability, but with this change, `i` deserves a better name. but since what we are actually using is but `i->second`, an alias for it would be better.